kit/themes/src/AnimateableText/AnimateableText.web.tsx

```diff
@@ -39,4 +39,4 @@ class AnimateableTextWeb extends React.Component<Text['props'] & { text: string
     }
 }
 
-export const AnimateableText = Animated.createAnimatedComponent(AnimateableTextWeb);
+export const AnimateableText: any = Animated.createAnimatedComponent(AnimateableTextWeb);
```

Comment:
What was the problem?

Reply:
Very weird TS error https://github.com/microsoft/TypeScript/issues/42873, I tried few things from the thread, but it didn't help, what's surprising is that `@ts-ingore` didn't work either (it's first time when I can't suppress an error). Since it has `.web` extension and the actual type is get from the file without extension it won't break anything, so I decided just suppress the error with `any` and don't mess with `tsconfig`

Reply:
But don't you loose an autocomplete here when developing for this component?

Reply:
Actual type goes from https://github.com/tonlabs/UIKit/blob/development/kit/themes/src/AnimateableText/AnimateableText.tsx#L3 , since TS doesn't understand suffixes on files :)